src/Statistics.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import StatisticLine from "./StatisticLine";
+
+const Statistics = ({ good, neutral, bad }) => {
+  const total = good + neutral + bad;
+  const average = total > 0 ? (good - bad) / total : 0;
```

Comment:
This statement (`total > 0`) is used twice.

FIX: extract into a constant or a method to be reused.